### ISSUE TITLE
Urgent hint not set on new chat message

### ISSUE THREAD

**Issue by ScoreUnder** [ValveSoftware/steam-for-linux#3670]
The urgent hint is not set on chat windows when a new message is received. This means that if I'm expecting a chat message I have to keep switching to the steam chat window to see if I have one, rather than simply see a red blip on steam's workspace (as I see with Skype, Hexchat and Pidgin).

**Comment by pyarmak** [comment]
Is there any news on this?
This is actually pretty annoying and I would like the urgent hint added as well. I use i3wm so I have to keep switching workspaces to check if I have messages.

**Comment by unknown3210** [comment]
Bump

**Comment by Ghosthree3** [comment]
Yeah I'm in the exact same boat as pyarmak. Very annoying, tend to go hours not noticing new messages.

**Comment by krompus** [comment]
Bump six years later. Still missing messages all the time on Linux because the window doesn't flash.

**Comment by Ghosthree3** [comment]
Since wayland has no urgency protocol and doesn't look like it will be getting one it's possible that this just isn't going to happen.
Unless something changes.
